TESTING.md

```diff
@@ -232,9 +232,20 @@ Sets the number of iterations each test uses by default.
 
 ### Running tests in the browser
 
-Normally we use `CLIENT=firefox` to run a set of tests in the browser
-automatically. This opens a browser window, automatically runs the requested
-tests in it, and reports the results back to the shell.
+To run tests in the browser, you first have to install playwright:
+
+```shell
+npx playwright install
+```
```

Comment:
i'm not sure this is the required command - ci is calling `npx playwright install-deps`:

https://github.com/pouchdb/pouchdb/blob/7d71f444e61cc6b4d5da14998a959bb62a0bfb09/.github/actions/install-playwright/action.yml#L7-L9

Reply:
Also does playwright already prompt for this or return a helpful error message if the dependencies aren't already available?  If that's the case, I wouldn't bother duplicating that documentation here.

Reply:
without this line, I get this error:

```
CLIENT=firefox npm test
…
Error starting tests: browserType.launch: Executable doesn't exist at /Users/jan/Library/Caches/ms-playwright/firefox-1440/firefox/Nightly.app/Contents/MacOS/firefox
╔═════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright Test or Playwright was just installed or updated. ║
║ Please run the following command to download new browsers:              ║
║                                                                         ║
║     npx playwright install                                              ║
║                                                                         ║
║ <3 Playwright Team                                                      ║
╚═════════════════════════════════════════════════════════════════════════╝
    at startTest (/Users/jan/Work/js/pouchdb/bin/test-browser.js:213:51)
    at checkReady (/Users/jan/Work/js/pouchdb/bin/dev-server.js:109:5)
    at Server.<anonymous> (/Users/jan/Work/js/pouchdb/bin/dev-server.js:103:5) {

```

so I’mma leave it